llvm/test/CodeGen/X86/tailcall-range.ll

```diff
@@ -0,0 +1,23 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=ipsccp -S -mtriple=x86_64-linux < %s | llc -mtriple=x86_64-linux | FileCheck %s
+
+define i32 @foo(ptr %this, ...) {
+; CHECK-LABEL: foo:
+; CHECK:       # %bb.0: # %entry
+; CHECK-NEXT:    movzbl 0, %eax
+; CHECK-NEXT:    retq
+entry:
+  %call = load volatile i1, ptr null, align 1
+  %spec.select = zext i1 %call to i32
+  ret i32 %spec.select
+}
+
+define i32 @bar(ptr %this, ...) {
+; CHECK-LABEL: bar:
+; CHECK:       # %bb.0: # %entry
+; CHECK-NEXT:    xorl %edi, %edi
+; CHECK-NEXT:    jmp foo@PLT # TAILCALL
+entry:
+  %4 = musttail call i32 (ptr, ...) @foo(ptr null, ...)
```

Comment:
The problem happen without the VA. Should I just remove the `...` in this test then?

Reply:
I think it's ok as long as they don't have relationship. I don't know how the range attribute been used.

Reply:
Remove them, tests are ideally as simple as possible demonstrating the issue

Reply:
think that it sounds good with simple test why not some thing like this instead of foo and bar
```
declare i32 @callee()

define range(i32 0, 2) i32 @func_with_range_attr() {
  %1 = musttail call i32 @callee()
  ret i32 %1
}

define i32 @call_with_range_attr() {
  %1 = musttail call range(i32 0, 2) i32 @callee()
  ret i32 %1
}
```